drivers/gpio/gpio_max32.c

```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright (c) 2023-2024 Analog Devices, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <pinctrl_soc.h>
```

Comment:
please do not include this header directly, it's meant to be internal.

Reply:
Thanks, removed.